lua/hbac/setup.lua

```diff
@@ -21,7 +21,8 @@ M.setup = function(opts)
 				buffer = 0,
 				once = true,
 				callback = function()
-					state.toggle_pin()
+					local bufnr = vim.api.nvim_get_current_buf()
+					state.toggle_pin(bufnr)
```

Comment:
Thank you for the fix, you were faster then me 😅

Reply:
I figured this was probably because in the old version you had `bufnr = bufnr or vim.api.nvim_get_current_buf` in the pinning func that was making passing bufnr as an arg unnecessary in some places. I didn't add it back in so there was no need to make further changes to your refactors in the notification function. This is cleaner overall imo